cypress/integration/multi_team/system_message_spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+/* eslint max-nested-callbacks: ["error", 4] */
+
+function verifySystemMessage(post) {
+    cy.get(post).
+        invoke('attr', 'class').
+        should('contain', 'post--system').
+        should('not.contain', 'same--root').
+        should('not.contain', 'other--root').
+        should('not.contain', 'current--user').
+        should('not.contain', 'post--comment').
+        should('not.contain', 'post--root');
+}
+
+function verifyStatusIconNotVisible(post) {
+    cy.get(post).
+        find('.status-wrapper .status svg').
+        should('not.be.visible');
+}
+
+describe('System Message', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('MM-15240 - No status on a system message', () => {
```

Comment:
Can you reorganize this to be two tests like this:

```
displayTypes.forEach(type => {
    it('has not status with ${type} display', () => {
        // test code
    }
})
```

Reply:
Done